src/test/java/org/opensearch/neuralsearch/search/query/HybridCollectorManagerTests.java

```diff
@@ -67,6 +68,7 @@ public class HybridCollectorManagerTests extends OpenSearchQueryTestCase {
     private static final String TEST_DOC_TEXT2 = "Hi to this place";
     private static final String TEST_DOC_TEXT3 = "We would like to welcome everyone";
     private static final String QUERY1 = "hello";
+    private static final String QUERY2 = "hi";
```

Comment:
Do not review this class as it already merged in main. The change is showing here because we have cherry picked the commit.